docker-compose-developer.yml

```diff
@@ -275,14 +275,12 @@ services:
   grafana:
     container_name: grafana
     labels: *oncall-labels
-    # image: "grafana/grafana:${GRAFANA_VERSION:-latest}"
-    image: "grafana/grafana-oss:9.3.2"
```

Comment:
fwiw, if you're trying to _easily_ modify the version of grafana running in docker-compose you can set the following env var in a root `.env` file 😄 
```.env
GRAFANA_VERSION=9.3.2
```
or a bit more manual, `GRAFANA_VERSION=9.3.2 make start`

Reply:
it was my mistake, I reverted changes